src/monodroid/jni/dylib-mono.h

```diff
@@ -109,6 +109,36 @@ typedef struct {
 	const unsigned int size;
 } MonoBundledAssembly;
 
+// Keep this in sync with MonoObject in object.h
+typedef struct {
+	// MonoObject 2 pointers
+	void *_pad_mono_object[2];
+} MonoObject;
```

Comment:
Why do we need these declarations? What's wrong with the prior `typedef void MonoObject`?